salt/roster/__init__.py

```diff
@@ -104,6 +104,7 @@ def targets(self, tgt, tgt_type):
             except salt.exceptions.SaltRenderError as exc:
                 log.error('Unable to render roster file: {0}'.format(exc))
             except IOError as exc:
+                log.error('Can\'t access roster for backend {0}: {1}'.format(back, exc))
```

Comment:
Single quotes aren't mandatory. And if you have them inside, please use double-quotes instead.